awx/main/migrations/_credentialtypes.py

```diff
@@ -197,3 +197,8 @@ def add_azure_cloud_environment_field(apps, schema_editor):
                                                    name='Microsoft Azure Resource Manager')
     azure_rm_credtype.inputs = CredentialType.defaults.get('azure_rm')().inputs
     azure_rm_credtype.save()
+
+def add_become_field(apps, schema_editor):
+    become_credtype = CredentialType.objects.get(kind='ssh')
```

Comment:
I would want to add the filter of `managed_by_tower` at least, in case we introduce custom ssh types or something like that